### ISSUE TITLE
Can i use Typescript for Cordova when execute instructions from Readme?

### ISSUE THREAD

**Issue by LightVolk** [dbiele/TypeScript-Cordova-SystemJS#1]
Can i use Typescript for Cordova when execute instructions from Readme?

Thank you!

**Comment by DeanB2015** [comment]
LightVolk, I'll be happy to help, but I'm not sure what you mean.  Can you clarify?

**Comment by LightVolk** [comment]
DeanB2015, i mean that i want to write code for Cordova using Angular 2 and Typescript. Is it possible, If i execute instructions from README file?

**Comment by DeanB2015** [comment]
Yes, it's possible to use the files to write code for Cordova with Angular 2 and TypeScript.  My idea was to create the code/template so any framework (angular2) using SystemJS will work.  

Note: I have not verified this with the beta version of Angular 2.  I'll verify the code works today and respond with any updates.  

I haven't had a lot of success using JSPM with PC and VS2015.  If you're interested, use the non-jspm version here:  https://github.com/dbiele/TypeScript-Cordova-SystemJS_no-JSPM Same code, just not using JSPM.

Thanks

**Comment by LightVolk** [comment]
Thank you! I will try.

**Comment by DeanB2015** [comment]
I tried adding the Angular2 Beta framework to the project and I'm running into problems. I believe there's a change in tsconfig.json using module = commonjs  & moduleresolution = node and possibly other issues.  

I'm confident I'll get a fix. I'll update the files as soon as I find a solution.  Of course, if you find anything, please use a pull request or contact me.

I used this simple code for starters:
https://angular.io/docs/ts/latest/quickstart.html

**Comment by DeanB2015** [comment]
LightVolk, I finally figured out the problem and created a new repository for the code.  There were a number of changes due to the changes in Angular2 Beta.  

Update project files : https://github.com/dbiele/TypeScript-Cordova-SystemJS-Angular2-beta 

I hope this helps.  Let me know if you have any problems.
